eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/SmoothFeeProvider.scala

```diff
@@ -34,6 +34,8 @@ class SmoothFeeProvider(provider: FeeProvider, windowSize: Int)(implicit ec: Exe
       _ = append(rate)
     } yield SmoothFeeProvider.smooth(queue)
   }
+
+  override def getName: String = provider.getName
```

Comment:
If we decide to keep provider's name, this should be more specific
```suggestion
  override def getName: String = s"Smooth(${provider.getName})"
```